tests/docker/python.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+ARG PRIVATE_KEY_FILE
```

Comment:
It is better to pass client id and scopes as parameters to command line rather than hardcoded env vars. Another option is to use docker run -e option to define them dynamically at the runtime.